lib/cli.js

```diff
@@ -142,7 +150,7 @@ exports.run = () => {
 		resolveTestsFrom: cli.input.length === 0 ? projectDir : process.cwd(),
 		projectDir,
 		timeout: conf.timeout,
-		concurrency: conf.concurrency ? parseInt(conf.concurrency, 10) : 0,
+		concurrency: conf.concurrency ? parseInt(conf.concurrency, 10) : 1,
```

Comment:
Oh sorry, I assumed the default had to change, because you wouldn't be able to set it to `0`.